src/SprykerEco/Zed/Payone/Business/Payment/DataMapper/ExpenseMapper.php

```diff
@@ -14,24 +14,35 @@
 
 class ExpenseMapper implements ExpenseMapperInterface
 {
+    /**
+     * @var string
+     */
     protected const HANDLING_PRODUCT_DESCRIPTION = 'Handling';
+
+    /**
+     * @var int
+     */
     protected const ZERRO_ITEM_TAX_RATE = 0;
+
+    /**
+     * @var int
+     */
     protected const ONE_ITEM_AMOUNT = 1;
 
     /**
-     * @param \Generated\Shared\Transfer\OrderTransfer $orderTransfer
+     * @param \Generated\Shared\Transfer\OrderTransfer|null $orderTransfer
      * @param \SprykerEco\Zed\Payone\Business\Api\Request\Container\AbstractRequestContainer $container
      *
      * @return \SprykerEco\Zed\Payone\Business\Api\Request\Container\AbstractRequestContainer
      */
-    public function mapExpenses(OrderTransfer $orderTransfer, AbstractRequestContainer $container): AbstractRequestContainer
+    public function mapExpenses(?OrderTransfer $orderTransfer, AbstractRequestContainer $container): AbstractRequestContainer
```

Comment:
several lines below you have `foreach ($orderTransfer->getExpenses(` 
that's a fatal error
not nullable

Reply:
Fixed.